migrationmanager/migrators/traces/migrations/000031_change_resource_index.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE signoz_traces.signoz_index_v2 ON CLUSTER {{.SIGNOZ_CLUSTER}}
+    DROP INDEX IF EXISTS idx_resourceTagsMapKeys,
+    DROP INDEX IF EXISTS idx_resourceTagsMapValues,
```

Comment:
Can you drop the `DROP INDEX` from this migration and use different name for tokenbf indexes? `DROP INDEX` is implemented as mutation and will likely fail the migration and lead to dirty migration issues.

Reply:
We can DROP INDEXES later when we have the schema migrator ready.

Reply:
the names are different. I changed to keep the name synced to the column name. New is `idx_resourceTagsMapKeys` an old was `idx_resourceTagMapKeys`. Notice the `Tags` vs `Tag`.

Do you want to remove the `DROP INDEX ...` from up file only or also removing `Add INDEX ...` in the down file?

Reply:
Ah, I didn't notice that. Let's just remove the `DROP INDEX` from up migration file.

Reply:
2 indexes on a column should be fine?

Reply:
Yes, it should be fine. We will incur some costs for index storage but from CH will pick the best one out of all created indexes.